File: Java/src/test/java/workshops/testingil/dirtytests/solutions/e09/more_tests/helpers/more_tests/helpers/SequenceMessageHelper.java
package workshops.testingil.dirtytests.solutions.e09.more_tests.helpers.more_tests.helpers;

public class SequenceMessageHelper {
    public String version;

    public char[] sequence;
    public boolean resetOnError;

    public void addSequence(String sequence) {
        this.sequence = sequence.toCharArray();
    }
}

File: CSharp/DirtyTestsWorkshop/DirtyTestsWorkshop/Exercise/SequenceMessageHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DirtyTestsWorkshop.Exercise
{
    public class SequenceMessageHelper
    {
        public string version;

        public char[] sequence;
        public bool resetOnError;

        public void addSequence(String sequence)
        {
            this.sequence = sequence.ToCharArray();
        }
    }
}
